src/pages/Cart/CartTotalSection.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './CartTotalSection.scss';
+
+class CartTotalSection extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // const { name, price, discount, image } = this.props.cartData;
+    const total = this.props.total;
+
+    return (
+      <section>
+        <div className="cartTotalSection">
+          <table className="cartTotalTable">
+            <thead>
+              <tr>
+                <th> 총 상품금액</th>
```

Comment:
```suggestion
                <th>총 상품금액</th>
```
띄어쓰기로 여백 주고 계신거 같은데 이런 부분은 스타일이기 때문에 margin, padding 사용해서 조절하는게 좋아보입니다

Reply:
제가 생각없이 띄어쓰기를 한 것 같습니다. 수정했습니다